src/pages/PortfolioSummary.tsx

```diff
@@ -0,0 +1,95 @@
+                    <IonText color="light">
+                        <h1 className="--font-bold heading-h1">Portfolio</h1>
+                    </IonText>
+                </div>
+                {/* Top Section */}
+                <div className="p-1">
+                    <div className="current-data-container">
+
+                    </div>
+                </div>
+                {/* End of Top Section */}
+                <div className="mb-2"></div>
+                {/* Holdings Section */}
+                <div className="p-1 pb-0">
+                    <IonText color="medium">
+                        <h6 className="--uppercase --small-text heading-h6">top 5 holdings</h6>
+                    </IonText>
+                </div>
+                <IonList class="holdings-list">
+                    {
+                        holdings.map((data, index) => {
+                            return (
+                                <IonItem class="holdings-list-item" key={index}>
+                                    <div className="w-100 p-1">
+                                        <IonGrid class="holdings-grid">
+                                            <IonRow>
+                                                <IonCol class="pl-0">
+                                                    <IonText color="medium">
+                                                        <h6 className="--small-text heading-h6">Invested:&nbsp;51009.00</h6>
+                                                    </IonText>
+                                                    <IonText color="light">
+                                                        <h6 className="--uppercase heading-h6">rcom</h6>
+                                                    </IonText>
+                                                    <IonText color="medium">
+                                                        <h6 className="--small-text heading-h6">Royalties</h6>
+                                                    </IonText>
+                                                </IonCol>
+                                                <IonCol></IonCol>
+                                                <IonCol class="pr-0">
+                                                    <IonText color="success">
+                                                        <h6 className="--small-text --text-right heading-h6">+15.36%</h6>
+                                                    </IonText>
+                                                    <IonText color="success">
+                                                        <h6 className="--text-right heading-h6">270.55</h6>
+                                                    </IonText>
+                                                    <p className="--small-text --text-right ">
+                                                        <IonText color="medium">
+                                                            +7.89&nbsp;
```

Comment:
need to take care of these numbers in proper way

Reply:
Yeah okay for the time being i have done like that, it will be like a proper value from json